components/brave_rewards/resources/extension/brave_rewards/components/panel.tsx

```diff
@@ -232,6 +232,10 @@ export class Panel extends React.Component<Props, State> {
         type = 'grant'
         text = getMessage('grantNotification')
         break
+      case 5:
```

Comment:
I know this isn't introduced here, but we're using a lot of magic numbers in this area of the code, could we define these as constants somewhere?

Reply:
@bbondy Yes, will do that now.